src/client/components/evaluate/detail/EqualPopulation.tsx

```diff
@@ -88,11 +88,11 @@ const EqualPopulationMetricDetail = ({
   return (
     <Box>
       <Heading as="h2" sx={{ variant: "text.h5", mt: 4 }}>
-        {metric.value?.toString() || " "} of the {metric.total} districts are within{" "}
+        {metric.value.toString() || " "} of {metric.total} districts are within{" "}
```

Comment:
This should fix the lint
```suggestion
        {metric.value?.toString() || " "} of {metric.total} districts are within{" "}
```